devolving_music/models/song_comparison.py

```diff
@@ -9,3 +9,12 @@ class SongComparison(models.Model):
     first_better = models.BooleanField()
     first_peakier = models.BooleanField()
     first_post_peakier = models.BooleanField()
+    
+    @staticmethod
+    def get_event_comparisons(Event):
+        comparison_event_submissions = [
+            sub
+            for sub in SongComparison.objects.select_related('first_submission').filter(first_submission__event__exact=Event).order_by('created_at')    
+        ]
```

Comment:
This doesn't need to be a comprehension since there's no transformation or filter. Just use `list()`